sdk/sdktypes/consts.go

```diff
@@ -0,0 +1,9 @@
+package sdktypes
+
+const (
```

Comment:
these are all internal consts that belong in fixtures, not here.

Reply:
why fixtures? 
those consts used in manifest plan/exec, triggers, cmd and scheduler. Do you want me to put them in fixture file?

Reply:
1. I moved builtinSchedulerConnectionID to `sdktypes/connection_id`. I even cannot keep it in fixtures, since `Strict()` uses it to check